linera-core/src/client.rs

```diff
@@ -192,6 +195,8 @@ impl MessagePolicy {
 /// * As a rule, operations are considered successful (and communication may stop) when
 /// they succeeded in gathering a quorum of responses.
 pub struct ChainClient<ValidatorNodeProvider, Storage> {
+    /// The Linera [`Client`] that manages operations on this chain.
+    client: Arc<Client<ValidatorNodeProvider, Storage>>,
```

Comment:
"on this chain" is a bit confusing to me. It's more wallet related, isn't it?

Reply:
I'm a bit confused by this comment — there's no wallet here yet (the wallet will be introduced in `linera-client`).  This is the protocol client that acts as a backend for the operations on this chain client.